File: src/main/java/com/github/izerui/file/entity/DeployEntity.java
package com.github.izerui.file.entity;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.util.Date;

/**
 * Created by serv on 2017/4/8.
 */
@Entity
@Table(name = "deploy")
public class DeployEntity {

    @Id
    private String fileName;
    private Date deployTime;

    public String getFileName() {
        return fileName;
    }

    public void setFileName(String fileName) {
        this.fileName = fileName;
    }

    public Date getDeployTime() {
        return deployTime;
    }

    public void setDeployTime(Date deployTime) {
        this.deployTime = deployTime;
    }
}

File: src/main/java/com/github/izerui/file/entity/DeployEntity.kt
package com.github.izerui.file.entity

import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table
import java.util.Date

/**
 * Created by serv on 2017/4/8.
 */
@Entity
@Table(name = "deploy")
class DeployEntity {

    @Id
    var fileName: String? = null
    var deployTime: Date = Date()
}
